default.env

```diff
@@ -75,3 +75,15 @@ export KLOG_VERBOSITY='1'
 # Example: 5d
 # Required: no
 export IMAGE_TAG_EXPIRATION='6h'
+
+# A valid Github App ID
+# Required: no
+PAC_GITHUB_APP_ID
```

Comment:
as env file it should be importable by bash, there should be `=`

Reply:
fixed.